tests-utils/src/main/scala/tech/beshu/ror/utils/elasticsearch/SearchManager.scala

```diff
@@ -34,14 +34,21 @@ class SearchManager(client: RestClient,
                     override val additionalHeaders: Map[String, String] = Map.empty)
   extends BaseManager(client, esVersion, esNativeApi = true) {
 
-  def search(indexName: String, query: JSON): SearchResult =
-    call(createSearchRequest(Some(indexName), query), new SearchResult(_))
+  def search(): SearchResult = search(List.empty)
+
+  def search(indexName: String): SearchResult = search(indexName :: Nil)
+
+  def search(indexNames: List[String]): SearchResult =
+    call(createSearchRequest(indexNames), new SearchResult(_))
 
   def search(query: JSON): SearchResult =
     call(createSearchRequest(None, query), new SearchResult(_))
 
-  def search(indexNames: String*): SearchResult =
-    call(createSearchRequest(indexNames.toList), new SearchResult(_))
+  def search(indexName: String, query: JSON): SearchResult =
+    call(createSearchRequest(Some(indexName), query), new SearchResult(_))
```

Comment:
maybe we could just call ujson.write here, and avoid complicating `def createSearchRequest` ?